terraform/_versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   }
   required_version = ">= 1.3.7"
   required_providers {
-    azurerm  = "= 3.81.0"
+    azurerm  = "4.4.0"
```

Comment:
@himesh99 I would recommend you use `3.116.0` for `azurerm`, the v4 release has a large number of breaking changes that are yet to be tested or worked through.